src/hooks/useProjektinTiedottaminen.ts

```diff
@@ -0,0 +1,34 @@
+import { useMemo } from "react";
+import useApi from "./useApi";
+import { useProjekti } from "./useProjekti";
+import { API } from "@services/api/commonApi";
+import useSWR, { Fetcher, SWRConfiguration } from "swr";
+import { apiConfig, ProjektinTiedottaminen } from "@services/api";
+
+export type UseIsProjektinTilaOptions =
+  | SWRConfiguration<ProjektinTiedottaminen | null, any, Fetcher<ProjektinTiedottaminen | null>>
+  | undefined;
+
+const defaultOptions = {
+  revalidateOnFocus: true,
+  revalidateIfStale: true,
+  revalidateOnReconnect: true,
+  refreshInterval: 10000,
+};
+
+export const useProjektinTiedottaminen = (_config: UseIsProjektinTilaOptions = {}) => {
+  const config: UseIsProjektinTilaOptions = useMemo(() => ({ ...defaultOptions, ..._config }), [_config]);
+  const api = useApi();
+  const { data: projekti } = useProjekti();
```

Comment:
voisko ton projektin oid:in hakea samalla tavalla kuin useProjekti:ssa kun ainoastaan oid tarvitaan?